### PR TITLE
write audit record to Authenticate database

### PR DESCRIPTION
**Pull request by csaltmountain**

### CODE REVIEW

**Review by jack-mccabe-defra — APPROVED** [review]
LGTM

File: app/data-sources/entra-id/EntraIdApi.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          },
+          cacheOptions: {
+            ttl: process.env.ENTRA_ID_TTL_IN_SECONDS
+          }
+        }
+      )
+
+      employeeId = response.employeeId
+    } catch (err) {
+      this.logger.error(err)
```
Comment (rkdefra) [inline app/data-sources/entra-id/EntraIdApi.js:29]: @csaltmountain I think this line is a problem! Has the logger actually been setup anywhere!?

**Review by rkdefra — COMMENTED** [review]
Possible `logger` issue!

File: app/data-sources/entra-id/EntraIdApi.js (hunk 1 above)
Comment (rkdefra) [reply app/data-sources/entra-id/EntraIdApi.js:29]: Fixed by #66

**Pull request merged**